Whiskey/Tasks/CreateFile.ps1

```diff
@@ -0,0 +1,69 @@
+function New-WhiskeyFile
+{
+    [Whiskey.Task('CreateFile')]
+    [CmdletBinding()]
+    param(
+        [Whiskey.Tasks.ValidatePath(Mandatory,PathType='File',MustExist=$false,AllowAbsolute)]
+        [string[]]$Path,
+
+        [Whiskey.Context]$TaskContext,
+
+        [string]$Content,
+
+        [bool]$Force,
+
+        [bool]$Touch
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
+    if (-not $TaskContext)
+    {
+        Stop-WhiskeyTask -TaskContext $TaskContext -Message ('No task context provided.')
+        return
+    }
+
+    foreach($item in $Path)

```

Comment:
Organize this like this:

```
foreach( $item in $Path )
{
    if( -not (Test-Path -Path $item) )
    {
        # Create
    }
    elseif( (Test-Path -Path $path -PathType Container) )
    {
        # Fail because it's a dir
    }
    else
    {
        # It already exists, so change last write time if Touch is true
    }
}
```